markata_todoui/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 from markata_todoui.posts import Posts
 
-__version__ = "0.0.1"
```

Comment:
Notice that this was way behind... Might be a bug with bump2version config in setup.cfg? It's not exactly how I'm used to seeing it

Reply:
Well then i know my next PR